automation/check-patch.e2e-lifecycle-k8s.sh

```diff
@@ -27,7 +27,7 @@ main() {
         export RELEASES_DESELECTOR="${to_be_released}"
     else
         # Don't run all upgrade tests in regular PRs, stick to those released under HCO
-        export RELEASES_SELECTOR="{0.18.0,0.23.0,0.27.0,99.0.0}"
+        export RELEASES_SELECTOR="{0.18.0,0.23.0,0.27.7,99.0.0}"
```

Comment:
are spaces allowed here between each release tag ?

Reply:
Not in the current implementation AFAIK.